### PR TITLE
Update description

### PR DESCRIPTION
**Pull request by ethomson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Basic:
 
 ``` yaml
 steps:
-- uses: actions/checkout@master
+- uses: actions/checkout@v1
```
Comment (tclem) [inline README.md]:
```suggestion
- uses: actions/checkout@v2
```

File: README.md [hunk 2]
```diff
@@ -36,7 +36,7 @@ jobs:
         cabal: [ '2.0', '3.0' ]
     name: Haskell GHC ${{ matrix.ghc }} sample
     steps:
-      - uses: actions/checkout@master
+      - uses: actions/checkout@v1
```
Comment (tclem) [inline README.md]:
```suggestion
      - uses: actions/checkout@v2
```

**Comment by fosskers** [comment]
Can this go in?

**Comment by fosskers** [comment]
Cool :+1:

**Pull request merged**